packages/apps/job-launcher/server/src/modules/payment/payment.service.spec.ts

```diff
@@ -1529,4 +1529,90 @@ describe('PaymentService', () => {
       );
     });
   });
+
+  describe('createWithdrawalPayment', () => {
+    it('should create a withdrawal payment successfully', async () => {
+      const userId = 1;
+      const amount = 100;
+      const currency = PaymentCurrency.USD;
+      const rate = 1;
```

Comment:
these variables are used for every test case, we can define them globally inside this `describe`